src/export/digitalPlanning/model.ts

```diff
@@ -415,7 +432,7 @@ export class DigitalPlanning {
                         source:
                           key === "road.classified"
                             ? "https://www.ordnancesurvey.co.uk/products/os-mastermap-highways-network-roads"
-                            : `https://planinng.data.gov.uk/entity/${entity.id}`,
+                            : `https://planinng.data.gov.uk/entity/${entity.entity}`,
```

Comment:
this was a small mapping bug in previous versions, leading to links like `https://planning.data.gov.uk/entity/undefined`